doc/source/whatsnew/v0.25.1.rst

```diff
@@ -85,6 +84,7 @@ Indexing
 - Bug in partial-string indexing returning a NumPy array rather than a ``Series`` when indexing with a scalar like ``.loc['2015']`` (:issue:`27516`)
 - Break reference cycle involving :class:`Index` and other index classes to allow garbage collection of index objects without running the GC. (:issue:`27585`, :issue:`27840`)
 - Fix regression in assigning values to a single column of a DataFrame with a ``MultiIndex`` columns (:issue:`27841`).
+- Fix assignment of column via `.loc` with numpy `non-ns datetime` type (:issue: `27395`)
```

Comment:
```suggestion
- Fix assignment of column via `.loc` with numpy `non-ns datetime` type (:issue:`27395`)
```

I'm not sure if sphinx will complain about the space or not.

Also, I think the backticks on "non-ns datetime" aren't necessary.